Source/DafnyCore/Auditor/Assumption.cs

```diff
@@ -0,0 +1,79 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+
+namespace Microsoft.Dafny.Auditor;
+
+public record AssumptionDescription(string issue, string mitigation, bool isExplicit) {
+  public static AssumptionDescription AxiomAttributeAssumption = new(
```

Comment:
I would inline these statics.

Reply:
I really like having them all in one place, so they can easily be reviewed for wording. Large string constants embedded in definitions seem like code smells to me.